nbformat/sign.py

```diff
@@ -123,8 +123,17 @@ def _db_default(self):
             self.log.warn("Missing SQLite3, all notebooks will be untrusted!")
             return
         kwargs = dict(detect_types=sqlite3.PARSE_DECLTYPES|sqlite3.PARSE_COLNAMES)
-        db = sqlite3.connect(self.db_file, **kwargs)
-        self.init_db(db)
+        try:
+            db = sqlite3.connect(self.db_file, **kwargs)
+            self.init_db(db)
+        except sqlite3.DatabaseError:
+            self.log.warn("Issues opening database: Moving the .db file to .db.bak and creating a new one!")
```

Comment:
I can a note that says "use jupyter --data-dir to find the old file."  That seems to be ungainly though - what if they don't use jupyter, or maybe what if that command is deprecated down the line?  Do you have a different suggestion?